components/NavigationBar/src/MDCNavigationBar.h

```diff
@@ -193,7 +186,25 @@ IB_DESIGNABLE
 /* Equivalent to leadingItemsSupplementBackButton. */
 @property(nonatomic) BOOL leftItemsSupplementBackButton;
 
+#pragma mark - Deprecated
+
 /** The text alignment of the navigation bar title. Defaults to NSTextAlignmentLeft. */
 @property(nonatomic) NSTextAlignment textAlignment __deprecated_msg("Use titleAlignment instead.");
 
+/**
+ Display attributes for the titleView's title text.
+
+ Setting this property will render an NSAttributedString with the assigned attributes across the
+ entire text.
+ */
+#if defined(__IPHONE_11_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_11_0)
+@property(nonatomic, copy, nullable)
+    NSDictionary<NSAttributedStringKey, id> *titleTextAttributes UI_APPEARANCE_SELECTOR
+    __deprecated_msg("Use titleFont instead");
+#else
+@property(nonatomic, copy, nullable)
+    NSDictionary<NSString *, id> *titleTextAttributes UI_APPEARANCE_SELECTOR
+    __deprecated_msg("Use titleFont instead");
+#endif
```

Comment:
This API is presently being used by internal clients - please follow the [deprecation policy](https://github.com/material-components/material-components-ios/blob/develop/contributing/deprecation_policy.md) here.

tl;dr: Only deprecate APIs once you've defined a deprecation plan and timeline. For now, please revert these deprecations in this PR.

Reply:
Alright I created an issue. #3224